### ISSUE TITLE
Figure out how to handle IERS downloads

### ISSUE THREAD

**Issue by bgottula** [seeing-things/track#134]
Astropy uses IERS data for certain operations such as determining sidereal time. This is necessary for good accuracy due to the unpredictable nature of Earth's polar motion and the evolution of the length of the day. Astropy will try to automatically download new data when needed but can fail with an exception if the system is offline. Find a way to handle this that ensures we have reasonably fresh data downloaded (perhaps with some kind of systemd service) when network is available but that won't cause the code to crash when network is unavailable. A warning might be okay, but crashing is definitely not okay. We also don't want any calls to astropy to pause at any point to even try to download stuff whenever it feels like it.

Worth noting that in our application stale IERS data is probably fine. Our pointing accuracy is almost certainly limited by other effects, and our alignment procedure should compensate for any small errors caused by Earth polar motion.

Example of an operation previously in model.py that could fail:
```
location = EarthLocation(lat=12*u.deg, lon=-13*u.deg, height=0*u.m)
t = Time(observation.unix_timestamp, format='unix', location=location)

# this will fail
t.sidereal_time('mean')
```

The above code is no longer present in that exact form, but we still have transformations between 'altaz' frame and 'icrs' frame and these transformations use IERS data.

References:
https://docs.astropy.org/en/stable/utils/iers.html
http://docs.astropy.org/en/stable/api/astropy.utils.iers.IERS_Auto.html

For testing purposes I found this short script to be useful to clear the IERS cache:
```
#!/usr/bin/env python3

from astropy.utils.data import clear_download_cache
clear_download_cache()
```

**Comment by bgottula** [comment]
@jgottula would be great to get your help with this. See https://github.com/seeing-things/track/blob/abstract-camera-class/track/model.py#L24-L30  for what I've done about this so far. One thing I don't like about this approach is that it causes all of the programs to take longer to start up (at least, I'm pretty sure this is the root cause) and `import track` from an interpreter also takes a couple seconds (probably because something imported in `__init__.py` is importing `model.py`). I think I'd rather just never try to automatically download IERS in `track` and rely on an external mechanism to do it.

**Comment by bgottula** [comment]
Started getting thMoving to position 4 of 13: Az: 202.50, Alt: 41.81

```
Converged on the target position. Attempting plate solving.
	Plate solver attempt 1 of 3...WARNING: Tried to get polar motions for times after IERS data is valid. Defaulting to polar motion from the 50-yr mean for those. This may affect precision at the 10s of arcsec level [astropy.coordinates.builtin_frames.utils]
Solution found!
e following warnings, probably related to this: 
```

**Comment by bgottula** [comment]
Got related warnings again today. This is one of the first times I've taken it out after a fresh OS install.

```
	Plate solver attempt 1 of 3...WARNING: failed to download ftp://anonymous:mail%40astropy.org@gdc.cddis.eosdis.nasa.gov/pub/products/iers/finals2000A.all and https://datacenter.iers.org/data/9/finals2000A.all, using local IERS-B: <urlopen error Unable to open any source! Exceptions were {'ftp://anonymous:mail%40astropy.org@gdc.cddis.eosdis.nasa.gov/pub/products/iers/finals2000A.all': URLError(gaierror(-2, 'Name or service not known')), 'https://datacenter.iers.org/data/9/finals2000A.all': URLError(gaierror(-2, 'Name or service not known'))}> [astropy.utils.iers.iers]
WARNING: (some) times are outside of range covered by IERS table. Assuming UT1-UTC=0 for coordinate transformations. [astropy.coordinates.builtin_frames.utils]
WARNING: failed to download ftp://anonymous:mail%40astropy.org@gdc.cddis.eosdis.nasa.gov/pub/products/iers/finals2000A.all and https://datacenter.iers.org/data/9/finals2000A.all, using local IERS-B: <urlopen error Unable to open any source! Exceptions were {'ftp://anonymous:mail%40astropy.org@gdc.cddis.eosdis.nasa.gov/pub/products/iers/finals2000A.all': URLError(gaierror(-2, 'Name or service not known')), 'https://datacenter.iers.org/data/9/finals2000A.all': URLError(gaierror(-2, 'Name or service not known'))}> [astropy.utils.iers.iers]
WARNING: Tried to get polar motions for times after IERS data is valid. Defaulting to polar motion from the 50-yr mean for those. This may affect precision at the arcsec level [astropy.coordinates.builtin_frames.utils]
Solution found!
```

**Comment by bgottula** [comment]
While working on this I noticed that Astropy also downloads a file with leap second data automatically. As an example, from the root of a virtual environment where Astropy is installed,

```
venv/lib/python3.8/site-packages/astropy/utils/iers/data/Leap_Second.dat
```

Unlike the IERS data this file does not get removed when running

```
from astropy.utils.data import clear_download_cache
clear_download_cache()
```

**Comment by bgottula** [comment]
I have disabled strict checking for stale IERS data in this project so that it should be able to proceed without an internet connection even if the data is a bit stale. Fresh IERS data is now downloaded as part of package installation.

Addressed in c704018f6ac7eee2abd773adfda7a82f252ee061
